clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/spring/client/properties/PropertyBasedJobWorkerValueCustomizer.java

```diff
@@ -212,4 +246,9 @@ private void applyDefaultJobWorkerTenantIds(final JobWorkerValue jobWorkerValue)
       }
     }
   }
+
+  private enum OverrideSource {
+    global,
```

Comment:
❓ if I got it correctly `global` == "client.worker.defaults", `worker` == "client.worker.override.<jobType>"? Shall we rename `global` ==> `defaults` then?

Reply:
Which actually makes we question the naming `defaults` 😅 , if I got it correctly whatever I set in `client.worker.defaults` will win over what is set on the annotation, right? So it's rather a global override than a default?

Reply:
Reading the `customize` method there are some exceptions though where the annotation wins, I find this hard to understand. Might it be worth separating `defaults` from `global_overrides`? I fear this might be a breaking change though :/

Reply:
>Shall we rename global ==> defaults then?

I agree, we should do that.

Reply:
>I agree, we should do that.

Done